### ISSUE TITLE
debug mode

### ISSUE THREAD

**Issue by joehobson** [navnorth/wp-oer#19]
Add a global debug mode, with a checkbox in the Settings page, similar to how we did it with the OII content-mirror plugin. Then put some debug on the screenshot feature so we can see what's going on there. I removed the comments on the xvfb portion on the test server (see #17) but it's still not getting screenshots correctly. Need better debug so we know what's going on.

It would also be helpful to add some debug on standards, category, and resource imports if those fail.

**Comment by johnpaulbalagolan** [comment]
fixed #19 with latest source code committed to github

**Comment by joehobson** [comment]
could be done cleaner. add a helper function for debug_log() so that we can use it in the code wherever without having to check the debug setting all over the place. Simplify it, so this: 
```
global $_debug;
if ($_debug=="on")
    error_log("OER Categories Importer: Initializing Excel Reader");
```

can just be:
```debug_log("OER Categories Importer: Initializing Excel Reader");```

**Comment by johnpaulbalagolan** [comment]
Fixed #19 with latest code committed to github
